### PR TITLE
Added environment variable for travis

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Partially solves #150

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Added environment variable so that phantomJS only triggered in TravisCI environment

### CODE REVIEW

File: app/tests/test.py [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 import os
 
+is_travis = 'TRAVIS' in os.environ
 class BadgeyayTest(unittest.TestCase):
 
 	@classmethod
 	def setUpClass(cls):
-		cls.driver = webdriver.PhantomJS()
+		if is_travis:
+			cls.driver = webdriver.Firefox()
```
Comment (srungta) [inline app/tests/test.py]: `If is_travis == true`
then it should be using phantom JS.
I think the conditions are reversed unless I am missing something.

Comment (jajodiaraghav) [reply app/tests/test.py]: Yes. I too think this is a mistake 👍

Comment (vaibhavsingh97) [reply app/tests/test.py]: @jajodiaraghav @srungta It's correct 😄 
Please see [this](https://stackoverflow.com/questions/2020598/in-python-how-should-i-test-if-a-variable-is-none-true-or-false).
It's just i have to swap the statements, as if travis environment is true than phantomJS should execute else Firefox driver should execute

File: app/tests/test.py [hunk 2]
```diff
@@ -3,11 +3,15 @@
 import time
 import os
 
+is_travis = 'TRAVIS' in os.environ
```
Comment (srungta) [inline app/tests/test.py]: Remove trailing whitespaces. Multiple places in this file

Comment (vaibhavsingh97) [reply app/tests/test.py]: @srungta can you please specify line number you are talking about 😄

Comment (vaibhavsingh97) [reply app/tests/test.py]: @srungta I checked the whole file and it's not of autopep8 standard. If we go through autopep8standard than i have to add more space. Please guide me if i miss something

Comment (jajodiaraghav) [reply app/tests/test.py]: @vaibhavsingh97 There is some whitespace after `os.environ`, can you please remove it?
Thanks

Comment (srungta) [reply app/tests/test.py]: The white space is giving you the "Not so good... This pull request quality could be better." in codacy. 
https://www.codacy.com/app/fossasia/badgeyay/pullRequest?prid=935773

Comment (vaibhavsingh97) [reply app/tests/test.py]: Ohh! I got it

**Comment by jajodiaraghav** [comment]
Thanks @vaibhavsingh97!

**Comment by srungta** [comment]
@jajodiaraghav  Should this be enough to close #150 ?

**Comment by jajodiaraghav** [comment]
@srungta No, We just need this documented then we can close this

**Pull request merged**